extensions/amp-flying-carpet/0.1/amp-flying-carpet.css

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Non-overridable properties */
+amp-flying-carpet {
+  position: relative !important;
+}
+
+/* Clip the contents of the flying-carpet so it is only viewable "inside" */
+amp-flying-carpet > .-amp-flying-carpet-clip {
+  /* Absolutely so clipping works */
+  position: absolute !important;
+  width: 100% !important;
+  height: 100% !important;
+  /*
+   * Use the deprecated clip style for browsers that do not support clip-path.
+   * Basically everything supports this, but we include clip-path to
+   * future-proof.
+   */
+  clip: rect(0, auto, auto, 0) !important;
+  clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
+  -webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
```

Comment:
we use autoprefixer when we transform the .css files into a .js file (during `gulp build`) here https://github.com/ampproject/amphtml/blob/master/build-system/tasks/jsify-css.js#L55